src/components/VCSMenu.tsx

```diff
@@ -1,7 +1,7 @@
 // Dependencies
 import * as React from "react";
-import { Button, Card, CardBody } from "reactstrap";
-
+import { Alert, Button, ButtonGroup, Card, CardBody, Collapse, CustomInput, Modal, ModalHeader, ModalBody, ModalFooter, Form, FormGroup, Input, Label } from "reactstrap";
```

Comment:
Its good to remove any unnecessary imports and leave them only in the file they're needed. For now, I'll remove them in the next PR after this one is merged.